collaborative_filtering/evaluation/__main__.py

```diff
@@ -0,0 +1,65 @@
+import sys
+import site
+import os
+
+cwd = sys.path[0]
+cwd = os.path.abspath(cwd)
+collaborative_filtering_dir = os.path.join(cwd, "..")
+site.addsitedir(collaborative_filtering_dir)
+
+import numpy as np
+import threading
+
+from evaluation import accurancy as ac
+from evaluation import selection
+from input import filesystem
+import similarity
+
+ratings_matrix = filesystem.read_ratings_matrix() # movie x user matrix
+#is_rated_matrix = filesystem.read_is_rated_matrix()
+is_rated_matrix = (ratings_matrix != 0)
```

Comment:
You could use the actual Y.csv file to be independent of whether zero is a possible rating. Why did you switch to this way instead of using the commented line above?

Reply:
oh yeah, forgot to undo that. Is fixed now